python/nx-cugraph/scripts/update_readme.py

```diff
@@ -190,14 +193,36 @@ def get_payload_internal(keys):
     return text
 
 
+def find_or_download_objs_file(objs_file_dir):
+    """
+    Returns the path to <objs_file_dir>/objects.inv, downloading it from
+    _objs_file_url if it does not already exist.
+    """
+    objs_file_path = objs_file_dir / "objects.inv"
+    if not objs_file_path.exists():
+        request = urllib.request.Request(_objs_file_url)
+        with urllib.request.urlopen(request) as response:
+            with open(objs_file_path, "wb") as out:
+                out.write(response.read())
+    return objs_file_path
+
+
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
         "Update README.md to show NetworkX functions implemented by nx-cugraph"
     )
     parser.add_argument("readme_filename", help="Path to the README.md file")
     parser.add_argument(
-        "networkx_objects", help="Path to the objects.inv file from networkx docs"
+        "networkx_objects", nargs="?", default=None,
+        help="Path to the objects.inv file from networkx docs. Optional."
```

Comment:
Minor nit: it would be nice to say that it's optional b/c it will search for and download the file if necessary.

Reply:
Agreed, and I added text to explain the behavior better.